libnaboConfig.cmake.in

```diff
@@ -6,13 +6,7 @@
 # Compute paths
 get_filename_component(libnabo_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 set(libnabo_INCLUDE_DIRS "@libnabo_include_dirs@")
- 
-# Our library dependencies (contains definitions for IMPORTED targets)
```

Comment:
Ok, I've checked non-catkin with static and dynamic libnabo with linux gcc. But what about Darwin? Paul might have been motivated by using his Mac back then. I neither have the possibility to test on mac nor the experience to be confident.